### PR TITLE
Remove GitHub Organisation Requirement for Registration

### PR DESCRIPTION
**Pull request by ghickman**
This removes the requirement to be in the OpenSAFELY GitHub Org when signing up, shifting it to the views where the difference between [un]authenticated users are displayed.

This is a preparatory step towards adding a demo of the platform (with the expectations backend), and the very beginnings (more in intent rather than implementation) of roles based permissions.

### CODE REVIEW

**Review by evansd — APPROVED** [review]
Code-wise this looks good. My question is whether we can deploy this until we've got some kind of minimal approval process in place to stop spam registrations?

**Comment by ghickman** [comment]
Good point, I'll take a look at putting Users in another table at first.

**Pull request merged**
